src/pages/nomad/api-docs/[...page].tsx

```diff
@@ -0,0 +1,34 @@
+import { GetStaticPathsContext, GetStaticPathsResult } from 'next'
+import { ProductData } from 'types/products'
+import DocsView from 'views/docs-view'
+// product data
+import nomadData from 'data/nomad.json'
+// server
+import { getStaticGenerationFunctions } from 'views/docs-view/server'
+import { removeIndexPath } from 'lib/remove-index-path'
+
+const basePath = 'api-docs'
+const baseName = 'API'
+const product = nomadData as ProductData
+
+const { getStaticPaths: generatedGetStaticPaths, getStaticProps } =
```

Comment:
For pages that didn't previously have "custom docs landing" pages, have moved `[[...page]]` files to `[...page]` files.